bower.json

```diff
@@ -5,7 +5,9 @@
     ".travis.yml"
   ],
   "dependencies": {
-    "o-colors": ">=2.5.0 <4",
-    "o-hoverable": ">=0.1.1 <4"
+    "o-colors": "~3.2.0",
+    "o-hoverable": ">=0.1.1 <4",
+    "o-fonts": "^2.0.0",
+    "guss-typography": "^3.1.1"
```

Comment:
guss-typography is used to get typography variants from the matrix. It was written by Kaelig so we're OK with that :)